### DISCUSSION TITLE
License for how many computers?

### DISCUSSION THREAD

**Discussion by Ainson** [loongmxbt/AxMath#10] [General]
License for how many computers?

**Comment by loongmxbt** [comment]
Only for 1. If you change a PC, you can send us an email to get new license.

**Comment by Ainson** [comment]
Ok, got 3 then. One for me, 2 for the kid (home desktop and school laptop).

Using it with Xournal++. Extremely handy for school work. Cheers.

**Comment by lawnwezza** [comment]
Hello, my computer is having some issues and I want to format the Hard disk. Do I need a new license or should my previous license be okay? Thank you.
